src/main/java/com/arondor/arender/alfresco/content/transformer/ContentTransformARenderRestClient.java

```diff
@@ -42,13 +42,15 @@ public float getWeatherPerformance()
         }
         catch (Exception e)
         {
-            LOGGER.log(java.util.logging.Level.INFO, "Caught exception while trying to fetch weather score, is rendition server down?");
+            LOGGER.log(java.util.logging.Level.INFO, "Caught exception while trying to fetch weather score, is rendition server down?",e);
             return -1;
         }
     }
 
     public void setAddress(String address)
     {
+        ARenderContentTransformerWorker aRenderContentTransformerWorker=new ARenderContentTransformerWorker();
```

Comment:
You should not create this object yourself. It is created automatically by spring.